src/niswitch/examples/niswitch_channel_properties.py

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/python
```

Comment:
This example probably should be removed.
We will add an example that shows this when we have something compelling to show (i.e. NI-DCPower).
@Fladolcetta what do you think? Ir is there something interesting to do with NI-SWITCH?

Reply:
Removed